src/vmm/src/persist.rs

```diff
@@ -280,6 +275,29 @@ fn snapshot_memory_to_file(
     }
 }
 
+/// Validate the microVM version and translate it to its corresponding snapshot data format.
+pub fn snapshot_version_sanity_check(
```

Comment:
Nit: Naming is inconsistent with functionality. A better name would be `get_snapshot_data_version`.

Reply:
You are right.